package.json

```diff
@@ -17,6 +17,7 @@
   "author": "gitwrecked",
   "license": "ISC",
   "dependencies": {
+    "atob": "^2.0.3",
```

Comment:
when adding files to the package.json on install, I think we should go back and remove all version specifiers and leave it as the version we are currently using, I.E.  `"atob": "^2.0.3"` ==>  `"atob": "2.0.3"`

that way, any update from the vendor side won't automatically be pulled in and we won't risk our codebase randomly breaking because of a change from the vendor.